notebooks/Custom_Model_TFX.ipynb

```diff
@@ -1,1339 +1,1839 @@
 {
```

Comment:
<p>It'd a be good idea to write a comment to guide the users about how to obtain this <code>.json</code>.  I suppose <code>FIREBASE_GCS_BUKCET</code>  needs to be appended <code>appspot.com</code>?  If it's the case then make a note about that too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/Dual-Deployments-on-Vertex-AI/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>